scripts/test_install_package.sh

```diff
@@ -0,0 +1,5 @@
+modulesList=("openqaoa-core" "openqaoa-qiskit" "openqaoa-braket" "openqaoa-pyquil" "openqaoa-azure")
+
+for entry in "${modulesList[@]}"; do
+    pip install -i https://test.pypi.org/simple/ --no-deps $entry
```

Comment:
Fair point. I believe we might not need this script specifically for the users to use. 
The idea of this script was just to have a quick way to install the modules from `pip` . 

Technically the user who clones this repository will not be using this script since `pip install .` would install this repository properly.

But i agree, I think once we've finalised which scripts will be in the final revision for the package, we can do a usage file for them. (This would most likely be for developers only, so its exact location would be TBD)